addon/components/hyper-table/filters-renderers/date.js

```diff
@@ -50,7 +50,19 @@ export default Component.extend(FiltersRendererMixin, {
     'This Year': 'this_year'
   },
 
+  _handleFlatpickrClick(e) {
+    e.stopPropagation();
+  },
+
   actions: {
+    openedFlatpickr() {
+      document.querySelectorAll('.flatpickr-calendar').forEach((fc) => {
```

Comment:
Wouldn't it be possible to grab the target Element from the action's event ? It would prevent the forEach part and you could just trigger the event listener directly on the target Element ?